docs/1.0/design-system/frontend-installation.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: docs
-title: Frontend framework
+title: Frontend installation
 ---
 
-# Frontend framework
+# Frontend installation
 
 ## Install NPM
 
 This repository is distributed with npm. After installing node.js, and npm you can install the front-end developer kit using:
 
 ```
-$ npm install --save ca-ui
+$ npm install --save cabui
```

Comment:
at the moment, this needs to be `cadesignsystem` (we will change this once we finalise a name, but for testing purposes it's `cadesignsystem`. Hope that's ok, would you mind changing this @Lyndsey28 and then I can get this merged in. Ta